src/dataDisplay/tableBody.jsx

```diff
@@ -1,37 +1,56 @@
-'use strict';
-
-import React, { Component } from 'react';
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
+import React from 'react';
 
-class TableBody extends Component {
-    render() {
-        const {
-            className,
-            id,
-            style,
-        } = this.props;
-        const containerClasses = ClassNames(
-            'table-body',
-            className
-        );
-
-        return (
-            <tbody
-                className={containerClasses}
-                id={id}
-                style={style}
-            >
-                {this.props.children}
-            </tbody>
-        );
-    }
-}
-
-TableBody.propTypes = {
+const propTypes = {
+    /**
+     * The content of the TableBody
+     */
+    children: PropTypes.node,
+    /**
+     * Assign additional class names to TableBody.
+     */
     className: PropTypes.string,
+    /**
+     * The `id` of the TableBody.
+     */
     id: PropTypes.string,
     style: PropTypes.shape({}),
 };
 
+const defaultProps = {
+    children: null,
+    className: null,
+    id: null,
+    style: null,
+};
+
+const TableBody = React.forwardRef((props, ref) => {
```

Comment:
@morethanfire @War777 @fantaJinMode @IlyaRadinsky 
I think we should consider following the technique outlined here:
https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools

That way we can still identify components using `forwardRef()` within the React Dev Tools.

Thoughts...?
This does not have to block this PR, although we should maybe consider doing it for components already diffed in here.

Perhaps in a separate PR, we should do a pass on React CM UI (and HC as well) where we do this comprehensively for all such components, so we go back and retrofit existing cases, and then we adopt this practice going forward.

Reply:
Fixed.

Reply:
Nice!  Thank you sir!
You think we should consider adding a GitHub issue or some such to go through and do this in other places?